gc/base/MemorySubSpaceSemiSpace.cpp

```diff
@@ -540,10 +542,8 @@ MM_MemorySubSpaceSemiSpace::flip(MM_EnvironmentBase *env, Flip_step step)
 		MM_HeapLinkedFreeHeader *lastFreeEntry = getDefaultMemorySubSpace()->getMemoryPool()->getLastFreeEntry();
 		if (NULL != lastFreeEntry) {
 			lastFreeEntrySize = lastFreeEntry->getSize();
-			if (debug) {
-				omrtty_printf("tilt restore_tilt_after_percolate last free entry %llx size %llx\n",
-						lastFreeEntry, lastFreeEntrySize);
-			}
+			Trc_MM_MSSSS_flip_restore_tilt_after_percolate(env->getLanguageVMThread(),
```

Comment:
you can report the info at line 559 along with those with 572 and 579 (if I see correctly, the info will not change)

for those at 572 and 579 you can have one trace point (potentially with one additional parameter where you pass string "a adjusted", but that's not even important since we will report 2 of them and we will know that the second one is the adjusted one)